developer-docs/macos.md

```diff
@@ -58,13 +57,21 @@ brew install boost-python3
 brew install ceres-solver
 ```
 
+<aside class="notice">
+Note - You might need to run
+
+    ln -s /usr/local/opt/opencv@3/ /usr/local/opt/opencv
+when you got `Reason: image not found` error.
+</aside>
+
 ### Install `libuvc`
 ```
 git clone https://github.com/pupil-labs/libuvc
 cd libuvc
 mkdir build
 cd build
 cmake ..
+make LIBRARY_PATH=/usr/local/lib
```

Comment:
I think the recommended way is to pass this as argument to `cmake`. Please make the appropriate changes.